src/Faker/Generator.php

```diff
@@ -333,6 +330,21 @@ public function filePath()
         return $this->ext(Extension\FileExtension::class)->filePath();
     }
 
+    public function bloodType(): string
```

Comment:
@Nyholm shouldnt we mark these as experimental too for now? Since the fact that we now have some code duplication on the providers and core methods. This way people atleast won't rely on it

Reply:
Good question. 
These technically already exist, so if we added `@experimental` on these methods, it would be a BC break.